robyn/__main__.py

```diff
@@ -0,0 +1,39 @@
+import click
+import os
+from robyn import Robyn
+
+
+@click.command()
+@click.option(
+    "--project-name",
+    prompt="Enter project name",
+    default="Sample-project",
+    help="The name of your new Robyn project",
+)
+@click.option(
+    "--project-dir",
+    prompt="Enter project directory",
+    default=os.getcwd(),
+    help="The directory where your new Robyn project is created",
+)
```

Comment:
@Oluwaseun241 , thank you for the great work. 😄 

I believe we should replace the `click.option` with an `input` instead.

e.g. 

```
def create():
    project_name = input("Enter the project name: ")
    project_dir = input("Enter the name of the project directory: ")
    
    ....
```


This way we will be able to create a flow of question and answers rather than a cli tool where everyone needs to remember the flags.